src/objc_glue_stubs.d

```diff
@@ -1,33 +1,35 @@
 
-/* Compiler implementation of the D programming language
+/**
+ * Compiler implementation of the D programming language
  * Copyright (c) 2015 by Digital Mars
  * All Rights Reserved
  * written by Michel Fortin
  * http://www.digitalmars.com
  * Distributed under the Boost Software License, Version 1.0.
  * http://www.boost.org/LICENSE_1_0.txt
- * https://github.com/dlang/dmd/blob/master/src/objc_glue_stubs.c
+ * https://github.com/dlang/dmd/blob/master/src/_objc_glue_stubs.d
  */
 
-#include <assert.h>
-#include <stdio.h>
+import core.stdc.stdio : printf;
 
-class FuncDeclaration;
-class Type;
-class TypeFunction;
-struct elem;
+import ddmd.func;
+import ddmd.mtype;
```

Comment:
All I'm doing with this is translating to D, I'm not fixing it, refactoring it, etc. It's just a rote translation. Improvements come later.